src/CMakeLists.txt

```diff
@@ -46,6 +46,10 @@ configure_file("${PROJECT_SOURCE_DIR}/include/miopen/config.h.in" "${PROJECT_BIN
 # configure a header file to pass the CMake version settings to the source, and package the header files in the output archive
 configure_file( "${PROJECT_SOURCE_DIR}/include/miopen/version.h.in" "${PROJECT_BINARY_DIR}/include/miopen/version.h" )
 
+#Copy header to Build Dir
+configure_file( "${PROJECT_SOURCE_DIR}/include/miopen/miopen.h" "${PROJECT_BINARY_DIR}/include/miopen/miopen.h" )
```

Comment:
Can a `if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)` be put around this? When we remove this option then this configure_file should be removed as well.

Reply:
@arvindcheru to raise awareness of the above comment.

Reply:
Thanks for the comments from @pfultz2 , @junliume . Current implementation for supporting backward compatibility we have used the build directory in line 764 (below snippet)
=======
 if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)
  rocm_wrap_header_dir(
    ${PROJECT_BINARY_DIR}/include/miopen
========================
So to make available all headers we were copying the missing file "miopen.h" in line 50 above.
So As per comment from Paul, we have changed this line under if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY).